1.5/alpine/Dockerfile

```diff
@@ -5,9 +5,9 @@ ENV GOLANG_SRC_URL https://golang.org/dl/go$GOLANG_VERSION.src.tar.gz
 ENV GOLANG_SRC_SHA256 002acabce7ddc140d0d55891f9d4fcfbdd806b9332fb8b110c91bc91afb0bc93
 
 RUN set -ex \
+	&& apk add --no-cache ca-certificates \
```

Comment:
We should just make this a separate `RUN` above the `ENV` lines.  We are already downloading the apk cache twice and this will let us share the layer between the go versions and might even work for other images from `alpine:3.4`